pandas/io/stata.py

```diff
@@ -1555,9 +1537,50 @@ def _read_value_labels(self) -> None:
                 self._value_label_dict[labname][val[i]] = self._decode(
                     txt[off[i] : end]
                 )
+
             if self._format_version >= 117:
                 self._path_or_buf.read(6)  # </lbl>
+
+    def _read_old_value_labels(self) -> None:
+        while True:
```

Comment:
Small docstring (one-liner) here and above indicating the versions that this targets would help readability without having to dig deeper.

Reply:
Good idea - I have now added this.